m68k-amigaos.cmake

```diff
@@ -23,17 +23,27 @@ set(AMIGAOS3 1)
 set(M68K_COMPILER "Bebbo")
 
 # CPU
-set(M68K_CPU_TYPES "68000" "68010" "68020" "68040" "68060" "68080")
+set(M68K_CPU_TYPES "68000" "68010" "68020" "68040" "68060" "68080" "custom")
 set(M68K_CPU "68000" CACHE STRING "Target CPU model")
 set_property(CACHE M68K_CPU PROPERTY STRINGS ${M68K_CPU_TYPES})
+if(M68K_CPU STREQUAL "custom")
+	set(M68K_CPU_FLAG "")
+else()
+	set(M68K_CPU_FLAG "-m${M68K_CPU}")
+endif()
 
 # FPU
-set(M68K_FPU_TYPES "soft" "hard")
+set(M68K_FPU_TYPES "soft" "hard" "custom")
 set(M68K_FPU "soft" CACHE STRING "FPU type")
 set_property(CACHE M68K_FPU PROPERTY STRINGS ${M68K_FPU_TYPES})
+if(M68K_FPU STREQUAL "custom")
+	set(M68K_FPU_FLAG "")
+else()
+	set(M68K_FPU_FLAG "-m${M68K_FPU}-float")
```

Comment:
this whole if-else affair is for backwards compatibility, I guess?

Reply:
yup